File: Recursion_using_java/factorial.java
import java.util.Scanner;

public class factorial{

    static int fab(int n){

        if (n == 0 || n == 1)
            return 1;
        else
            return n *fab(n-1);

    }

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter Number :");
        int n = sc.nextInt();
        System.out.println("The factorial of " + n + " is :" + fab(n));
    }
}

File: factorial.c
#include <stdio.h>
int main()
{
  int n,pdt=1;
  printf("enter the number");
  scanf("%d",&n);
  for(int i=1;i<=n;i++)
    {
      pdt=pdt*i;
    }
  printf("The factorial value is %d",pdt);
  return 0;
}
